dapp/src/components/buySell/DisclaimerTooltip.js

```diff
@@ -34,7 +32,7 @@ const DisclaimerTooltip = ({
         {children && (
           <div
             className="d-flex align-items-center justify-content-center"
-            onClick={(e) => {
+            onMouseOver={(e) => {
```

Comment:
This will probably not work on mobile. Would be cool that it works in both envrionments

Reply:
Also looks like we haven't addressed the z-sorting well in the past: 
<img width="470" alt="Screenshot 2021-12-20 at 10 41 25" src="https://user-images.githubusercontent.com/579910/146746650-8a59bf84-6c56-4254-b209-fb3728f05e2a.png">

Reply:
Fixed UI issues, used onClick event for the tooltip, shows only one button as the status(swap or approve)